### PR TITLE
chore(docs): button page (VIV-2087)

### PR DESCRIPTION
**Pull request by TaylorJ76**

### CODE REVIEW

**Review by rachelbt — COMMENTED** [review]
Looks really good and makes it easy to focus on what is needed.
didn't went over all changes - but for now:
1. can you set the tabs to be with shape rounded and not sharp?
2. under code > install, I think web-components need to be first
3. under properties, can we ski[ the names of the firs icons in the gallery and only point there or something?
4. there are missing connotation and appearances

File: package-lock.json
Comment (RichardHelm) [inline package-lock.json:1]: Why did this file change in the first place? Maybe it causes the failures

File: apps/docs/content/_data/components.json [hunk 1]
```diff
@@ -1,279 +1,347 @@
...
 		"markdown": "./libs/components/src/lib/breadcrumb-item/README.md",
 		"parent": "Breadcrumb"
 	},
 	{
 		"title": "Button",
-		"markdown": "./libs/components/src/lib/button/README.md"
+		"description": "Buttons trigger an action, such as submitting a form, closing a dialog, or navigating to a new page.",
+		"variations": "./libs/components/src/lib/button/VARIATIONS.md",
+		"guidelines": "./libs/components/src/lib/button/GUIDELINES.md",
+		"hideGuidelines": "true",
```
[the remark this answers is not in the document]
Comment (TaylorJ76) [reply apps/docs/content/_data/components.json:46]: I want them to be agreed and updated by design before publishing them.

**Comment by RichardHelm** [comment]
@TaylorJ76 Build is failing now because the new Searchable Select is missing `"page": "legacy"` in `components.json`

**Review by rachelbt — COMMENTED** [review]
I'm taking back my comment regarding the full rounded border-radius becasue of this:
image
Either we roll back to 8px 8px 0 0; or move the selected border to n after pseudo element :)

**Pull request merged**